eicm_flows/compute_all_matrices.py

```diff
@@ -10,12 +10,13 @@
 from eicm_flows.shading_reference_yokogawa import Microscopes, \
     create_shading_reference_yokogawa
 
+GROUPS = Choices.load("fmi-groups").get()
 
 class RawData(BaseModel):
     input_dir: str = "/tungstenfs/scratch/gmicro/reitsabi/CV7000/Flatfield_correction_tests/20221221-Field-illumination-QC_20221221_143935/Dyes_60xW_Cellvis/"
     microscope: Microscopes = "CV7000"
     z_plane: int = 33
-    group: Choices.load("fmi-groups").get() = "gmicro"
+    group: GROUPS = GROUPS.gmicro
```

Comment:
This (and the previous version of the code as well) assumes that `gmicro` is a valid member of the enum `GROUPS` provided by the block. That's not safe, provided we can configure any values for the block in the UI, right?
@tibuch Is there a way to ask the block for a default/preferred value?

Reply:
Good point! 

I don't know. Maybe the block should implement a method `get_default()`?

Reply:
Hm, maybe just using `GROUPS(0)` is simpler and more general here.

https://stackoverflow.com/questions/44867597/is-there-a-way-to-specify-a-default-value-for-python-enums